heclib/heclib_c/src/Public/ztsOffset.c

```diff
@@ -108,17 +108,17 @@ int ztsOffset(int intervalSeconds, int julian, int seconds)
 	cleanTime(&jul, &secs, SECOND_GRANULARITY);
 
 
-	if (intervalSeconds <= 86400) {
+	if (intervalSeconds <= SECS_IN_1_DAY) {
 		// Daily or less (most common case)
 		number = secs / intervalSeconds;
 		offsetSeconds = secs - (number * intervalSeconds);
 		return offsetSeconds;
 	}
 
-	if (intervalSeconds == 604800) {
+	if (intervalSeconds == SECS_IN_1_WEEK) {
 		// Weekly (EOP is Saturday night at midnight)
 		day = dayOfWeek(jul);
-		offsetSeconds = ((day - 1) * 86400) + secs;
+		offsetSeconds = ((day - 1) * SECS_IN_1_DAY) + secs;
 		if (offsetSeconds == 604800) offsetSeconds = 0;
```

Comment:
604800 could be replaced with SECS_IN_1_WEEK

Reply:
Oops - not sure how I let that one escape.

Reply:
Done